src/main/java/Bot.java

```diff
@@ -0,0 +1,55 @@
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.Locale;
+
+public class Bot {
```

Comment:
Do not forget to add spacing between method and First curly braces.

Reply:
Resolved